example/integration_test/bucketeer_flutter_sdk_test.dart

```diff
@@ -351,4 +355,27 @@ void main() async {
           BKTClient.instance.removeEvaluationUpdateListener(listenToken);
         });
   });
+
+  test('BKTClient.initialize should allow feature_tag to be optional', () async {
```

Comment:
Please add a unit test and check if the tag is empty after initializing the SDK.

E.g.

```kt
expect(config.featureTag, '');
```